yarn.lock

```diff
@@ -4501,7 +4501,7 @@ commondir@^1.0.1:
   resolved "https://registry.yarnpkg.com/commondir/-/commondir-1.0.1.tgz#ddd800da0c66127393cca5950ea968a3aaf1253b"
   integrity sha1-3dgA2gxmEnOTzKWVDqloo6rxJTs=
 
-compare-versions@^3.5.1:
+compare-versions@^3.5.1, compare-versions@^3.6.0:
```

Comment:
```suggestion
compare-versions@^3.6.0:
```
We were using 3.6 anyway, no need to support both

Reply:
It's autogenerated every time I run `yarn`. I tried to remove `compare-versions@^3.5.1,` but it keeps coming back! 😅

Reply:
Never mind, it came down to my poor understanding of `yarn.lock` syntax. What is actually happening is `husky` requires `compare-versions "^3.5.1"` and this line basically says if you are looking for `compare-versions@^3.5.1`, or `compare-versions@^3.6.0`, then the following is the package you want.